health/health.c

```diff
@@ -509,32 +515,30 @@ SILENCE_TYPE check_silenced(RRDCALC *rc, char* host, SILENCERS *silencers) {
  * @return It returns 1 case rrdcalc_flags is DISABLED or 0 otherwise
  */
 int update_disabled_silenced(RRDHOST *host, RRDCALC *rc) {
-	uint32_t rrdcalc_flags_old = rc->rrdcalc_flags;
-	// Clear the flags
-	rc->rrdcalc_flags &= ~(RRDCALC_FLAG_DISABLED | RRDCALC_FLAG_SILENCED);
-	if (unlikely(silencers->all_alarms)) {
-		if (silencers->stype == STYPE_DISABLE_ALARMS) rc->rrdcalc_flags |= RRDCALC_FLAG_DISABLED;
-		else if (silencers->stype == STYPE_SILENCE_NOTIFICATIONS) rc->rrdcalc_flags |= RRDCALC_FLAG_SILENCED;
-	} else {
-		SILENCE_TYPE st = check_silenced(rc, host->hostname, silencers);
-		if (st == STYPE_DISABLE_ALARMS) rc->rrdcalc_flags |= RRDCALC_FLAG_DISABLED;
-		else if (st == STYPE_SILENCE_NOTIFICATIONS) rc->rrdcalc_flags |= RRDCALC_FLAG_SILENCED;
-	}
-
-	if (rrdcalc_flags_old != rc->rrdcalc_flags) {
-		info("Alarm silencing changed for host '%s' alarm '%s': Disabled %s->%s Silenced %s->%s",
-				host->hostname,
-				rc->name,
-			 (rrdcalc_flags_old & RRDCALC_FLAG_DISABLED)?"true":"false",
-			 (rc->rrdcalc_flags & RRDCALC_FLAG_DISABLED)?"true":"false",
-			 (rrdcalc_flags_old & RRDCALC_FLAG_SILENCED)?"true":"false",
-			 (rc->rrdcalc_flags & RRDCALC_FLAG_SILENCED)?"true":"false"
-		);
-	}
-	if (rc->rrdcalc_flags & RRDCALC_FLAG_DISABLED)
-		return 1;
-	else
-		return 0;
+    uint32_t rrdcalc_flags_old = rc->rrdcalc_flags;
+    // Clear the flags
+    rc->rrdcalc_flags &= ~(RRDCALC_FLAG_DISABLED | RRDCALC_FLAG_SILENCED);
+    if (unlikely(silencers->all_alarms)) {
+        if (silencers->stype == STYPE_DISABLE_ALARMS) rc->rrdcalc_flags |= RRDCALC_FLAG_DISABLED;
+        else if (silencers->stype == STYPE_SILENCE_NOTIFICATIONS) rc->rrdcalc_flags |= RRDCALC_FLAG_SILENCED;
+    } else {
+        SILENCE_TYPE st = check_silenced(rc, host->hostname, silencers);
+        if (st == STYPE_DISABLE_ALARMS) rc->rrdcalc_flags |= RRDCALC_FLAG_DISABLED;
+        else if (st == STYPE_SILENCE_NOTIFICATIONS) rc->rrdcalc_flags |= RRDCALC_FLAG_SILENCED;
+    }
+
+    if (rrdcalc_flags_old != rc->rrdcalc_flags) {
+        info("Alarm silencing changed for host '%s' alarm '%s': Disabled %s->%s Silenced %s->%s",
+             host->hostname,
+             rc->name,
+             (rrdcalc_flags_old & RRDCALC_FLAG_DISABLED)?"true":"false",
+             (rc->rrdcalc_flags & RRDCALC_FLAG_DISABLED)?"true":"false",
+             (rrdcalc_flags_old & RRDCALC_FLAG_SILENCED)?"true":"false",
+             (rc->rrdcalc_flags & RRDCALC_FLAG_SILENCED)?"true":"false"
+        );
+    }
+
+    return (rc->rrdcalc_flags & RRDCALC_FLAG_DISABLED)?1:0;
```

Comment:
I would leave this line as it was, since we're supposedly only doing tab changes in this PR. It may help align the rest of the lines as well, because the line numbers change due to this. I don't like the diff that github shows (probably just because of this line), it's misleading.

Reply:
You are correct, I return it for the previous code changing only the tabulation.